README.md

```diff
@@ -7,8 +7,10 @@
 [![npm version](https://badge.fury.io/js/emotion.svg)](https://badge.fury.io/js/emotion)
 [![Build Status](https://travis-ci.org/tkh44/emotion.svg?branch=master)](https://travis-ci.org/tkh44/emotion)
 [![codecov](https://codecov.io/gh/tkh44/emotion/branch/master/graph/badge.svg)](https://codecov.io/gh/tkh44/emotion)
-![gzip size](http://img.badgesize.io/https://unpkg.com/emotion/dist/DO-NOT-USE.min.js?compression=gzip&label=gzip%20size)
-![size](http://img.badgesize.io/https://unpkg.com/emotion/dist/DO-NOT-USE.min.js?label=size)
+![core gzip size](http://img.badgesize.io/https://unpkg.com/emotion/dist/DO-NOT-USE.min.js?compression=gzip&label=core%20gzip%20size)
+![core size](http://img.badgesize.io/https://unpkg.com/emotion/dist/DO-NOT-USE.min.js?label=core%20size)
+![react gzip size](http://img.badgesize.io/https://unpkg.com/emotion/dist/DO-NOT-USE-rect.min.js?compression=gzip&label=react%20gzip%20size)
+![react size](http://img.badgesize.io/https://unpkg.com/emotion/dist/DO-NOT-USE-react.min.js?label=react%20size)
```

Comment:
Don't worry about it for the pr. If it looks confusing we can edit the readme later.